ckan-base/2.10/Dockerfile

```diff
@@ -16,6 +16,10 @@ ENV CKAN__PLUGINS image_view text_view recline_view datastore datapusher envvars
 # UWSGI options
 ENV UWSGI_HARAKIRI=50
 
+# optional. If the "expire_api_token" is enabled you will need this.
+ENV DATAPUSHER_TOKEN_EXPIRES_IN=
```

Comment:
Can you add these two in the `.env.example` file so they are documented somewhere?

Reply:
Documented here https://github.com/okfn/docker-ckan/commit/49c4185465c4c7dd0e5d2526407e85a2fb011ce6 @amercader